/home/zcw7438/.local/lib/python3.6/site-packages/pandas/compat/_optional.py:106: UserWarning: Pandas requires version '1.2.1' or newer of 'bottleneck' (version '1.2.0' currently installed).
  warnings.warn(msg, UserWarning)
Traceback (most recent call last):
  File "AggregateToMonthOrQuarter_A.py", line 60, in <module>
    Limit_to_Top100_and_Aggregate_to_quarter_or_month(product, quarterOrmonth)
  File "AggregateToMonthOrQuarter_A.py", line 38, in Limit_to_Top100_and_Aggregate_to_quarter_or_month
    data_chunk['owner initial'] = data_chunk['brand_descr'].map(brand_owner['owner initial'])
  File "/home/zcw7438/.local/lib/python3.6/site-packages/pandas/core/series.py", line 3828, in map
    new_values = super()._map_values(arg, na_action=na_action)
  File "/home/zcw7438/.local/lib/python3.6/site-packages/pandas/core/base.py", line 1277, in _map_values
    indexer = mapper.index.get_indexer(values)
  File "/home/zcw7438/.local/lib/python3.6/site-packages/pandas/core/indexes/base.py", line 2985, in get_indexer
    "Reindexing only valid with uniquely" " valued Index objects"
pandas.core.indexes.base.InvalidIndexError: Reindexing only valid with uniquely valued Index objects
